plugin/builtin/helmchartinflationgenerator/HelmChartInflationGenerator.go

```diff
@@ -329,3 +334,12 @@ func (p *plugin) checkHelmVersion() error {
 	}
 	return nil
 }
+
+// chartPath will return the path to the chart and handle the case where a version
+// is specified
+func (p *plugin) chartPath() string {
+	if p.Version != "" {
+		return filepath.Join(p.absChartHome(), fmt.Sprintf("%s-%s", p.Name, p.Version))
```

Comment:
Instead of creating a new function `chartPath`, let's fold this if statement into `absChartHome` and have `absChartHome` return the path with the version.

Reply:
good call, made those changes here: https://github.com/kubernetes-sigs/kustomize/pull/4999/commits/380abb9e79ace1b03d15985e885e4b9e42c78ac7